client/src/app/tabs/rpa/__tests__/RPAEditorSpec.js

```diff
@@ -262,3 +300,30 @@ function renderEditor(xml, options = {}) {
     wrapper
   };
 }
+
+async function expectEventually(expectStatement) {
```

Comment:
Why don't we just use `waitFor`?

```
await waitFor(() => expect(true).to.be.false);
```

Reply:
changed in [1de90c2](https://github.com/camunda/camunda-modeler/pull/4944/commits/1de90c2712f0ba12063a30435e8c0f101a6279ad)